src/Providers/EventServiceProvider.php

```diff
@@ -50,5 +51,15 @@ public function boot()
              */
             app(MailerLiteServiceContract::class)->deleteSubscriber($event->getUser());
         });
+
+        Event::listen(AbandonedCartEvent::class, function ($event) {
```

Comment:
Should we add another event that removes user from this group if they complete their order / buy anything?

Reply:
i add your question to ticket https://escl24.atlassian.net/browse/STAR-473